packages/ckeditor5-email/src/integrations/table.ts

```diff
@@ -41,18 +41,22 @@ export default class TableEmailIntegration extends Plugin {
 	 * @inheritDoc
 	 */
 	public afterInit(): void {
-		this._checkPlainTableOutputPlugin();
+		this._checkRequiredTablePlugins();
 		this._checkTableConfig();
 	}
 
 	/**
 	 * Checks if the `PlainTableOutput` plugin is available when the `Table` plugin is enabled.
 	 */
-	private _checkPlainTableOutputPlugin(): void {
+	private _checkRequiredTablePlugins(): void {
```

Comment:
I think that description should also be changed to reflect what function does now.